src/main/java/org/json/XMLTokener.java

```diff
@@ -20,6 +20,8 @@ public class XMLTokener extends JSONTokener {
     */
    public static final java.util.HashMap<String, Character> entity;
 
+   private static XMLParserConfiguration configuration = XMLParserConfiguration.ORIGINAL;;
```

Comment:
If configuration must be static, then add a comment explaining why. Otherwise, it probably should not be static.

Reply:
Will update this